src/ssl/crtd_message.cc

```diff
@@ -128,9 +128,7 @@ void Ssl::CrtdMessage::setCode(std::string const & aCode) { code = aCode; }
 std::string Ssl::CrtdMessage::compose() const
 {
     if (code.empty()) return std::string();
-    char buffer[10];
-    snprintf(buffer, sizeof(buffer), "%zd", body.length());
-    return code + ' ' + buffer + ' ' + body;
+    return code + ' ' + std::to_string(body.length()) + ' ' + body;
```

Comment:
std::to_string is locale-sensitive and, hence, slow. However, I think this fix is the right step forward because this code is already exposed to various std::string problems, and the correct long-term solution requires complex refactoring (to get rid of std::string).

This comment does not request any PR changes.